src/i18n/resources/en/ui.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Wasn't sure where to add, introduced new `ui` translation file for UI components